lib/matplotlib/mpl-data/matplotlibrc

```diff
@@ -156,7 +156,7 @@
 ## ***************************************************************************
 ## * HATCHES                                                                 *
 ## ***************************************************************************
-#hatch.color:     black
+#hatch.color:     inherit
```

Comment:
This is an API break as people who are currently expecting the hatch to be black (and are happy with that) in collections but are setting the edge color are going to be broken.

Can you add a test that would have been broken by this change as part of this PR?

Reply:
The only case where people would expect a hatch to be black and it would not be after this change is when the user is changing the rcParam["patch.edgecolor"]. Do you mean this?

If yes, the fix would be to revert this change and not use inherit at all.

Reply:
I'm willing to accept the changed behavior for this very edge case (see https://github.com/matplotlib/matplotlib/pull/28104#issuecomment-2448609728)